spec/models/admin_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Admin, type: :model do
+=begin
+  # undefined method `encrypted_password=' for #<Admin:0x000000033dda80>
+  it 'should be valid' do
+    build(:admin).should be_valid
```

Comment:
@DmitrS please use `create` for models which use `Devise`.
Also please [don't use block comments](https://github.com/bbatsov/ruby-style-guide#no-block-comments).